dom.bs

```diff
@@ -511,6 +511,7 @@ interface Event {
   readonly attribute unsigned short eventPhase;
 
   void stopPropagation();
+           attribute boolean cancelBubble; // historical
```

Comment:
This is the normal indentation for attributes without readonly. They are aligned with their readonly counterparts.